binaries/apiserver/main.go

```diff
@@ -10,7 +11,9 @@ func main() {
 	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
 	transportFactory := thrift.NewTTransportFactory()
 
-	err := server.Serve(server.NewHandler(), "localhost:9090", transportFactory, protocolFactory)
+	queue, _ := memory.NewSimpleQueue()
```

Comment:
recommend adding a TODO here to swap out with durable implementation.  I've been doing that in the code, so it'll be easy to ensure we don't miss anything when we move to MVP